bitcoin-magazine/bitcoin-explained/bitcoin-core-0.21.0-new-release.md

```diff
@@ -0,0 +1,915 @@
+
+Yeah, so the first change, the mempool, right?
+
+It used to be that when you have a Bitcoin Core wallet, you would be broadcasting transactions from your wallet. And then after 15 seconds or so you would be broadcasting transactions from the wallet and you would just keep broadcasting it to your peers nearby.
+
+Aaron van Wirdum: 00:03:26
+
+Isn't it 15 minutes?
+
+Sjors Provost: 00:03:27
+
+Yeah 15 minutes. So it would very frequently broadcast its own transactions because the wallet really wants to make sure that the rest of the world hears about its transactions. The problem with that...
+
+Aaron van Wirdum: 00:03:37
+
+Until it's included in a block, of course.
+
+Sjors Provost: 00:03:40
+
+Exactly. The problem with that is it's not very good for privacy. Because if you can pinpoint which node is constantly repeating a transaction, then that suggests that that's the node that created it.
+
+Aaron van Wirdum: 00:03:51
+
+Yeah, it makes it very obvious which addresses, which inputs originate from which node, which is bad for privacy, because a node can be tied to an IP address and an IP address to an identity and so forth.
+
+Sjors Provost: 00:04:03
+
+Exactly. So the long-term goal here, which has not been achieved in this specific change, but we always do little micro steps.
+
+Aaron van Wirdum: 00:04:11
+
+Uh huh
+
+Sjors Provost: 00:04:11
+
+The long-term goal is to get to a place where the node will just treat its own transactions, its own wallet transactions, just like it would transactions from another peer. So it will not prioritize its own transactions. And there's some downside to that because you have to keep your wallet open probably a little bit longer to make sure that it does get broadcast. But yeah, then it would be indistinguishable. So I think one of the long-term goals is to say, we're going to wait until the wallet transaction fee is close to what it would be for the next block, because you can order transactions by fee.
+
+Aaron van Wirdum: 00:04:45
+
+Yeah, I think what you're trying to say is that right now this rebroadcasting is done blindly while in the future the plan is to actually see if the transaction should have been included in a block by now based on the fees and only if that's not the case will it rebroadcast the transaction and it will do so for any transaction the node has seen not just for its own transactions.
+
+Sjors Provost: 00:05:08
+
+Exactly and this also creates a better separation between because then you know Bitcoin Core is a node which has a mempool and it's a wallet and you know this dealing with mempools should be done by the node, and it shouldn't be done by the wallet. So there's a lot of process separation going on under the hood there.
+
+Aaron van Wirdum: 00:05:25
+
+Yeah, but this, what we're describing right now, is actually not included in Bitcoin Core 21 right now. We've seen one micro step towards this in Bitcoin Core 21.
+
+Sjors Provost: 00:05:26
+
+Yeah.
+
```

Comment:
Another example of a line that doesn't contribute to the transcript's content - you can just delete this and continue with the same speaker.